src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -451,6 +454,17 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         /// </remarks>
         public IssuerSigningKeyValidatorUsingConfiguration IssuerSigningKeyValidatorUsingConfiguration { get; set; }
 
+        /// <summary>
+        /// Gets a <see cref="IDictionary{String, Object}"/> that is unique to this instance.
+        /// Calling <see cref="Clone"/> will result in a new instance of this IDictionary.
+        /// </summary>
+        public IDictionary<string, object> InstancePropertyBag { get; } = new Dictionary<string, object>();
```

Comment:
>get [](http://example.com/codeflow?start=65&length=3)

What do you think about throwing an exception (or at least logging a warning) if someone calls get when IsClone == false? That's never valid correct?

Reply:
Good for further discussion.